### DISCUSSION TITLE
Sprint 2 Retrospective

### DISCUSSION THREAD

**Discussion by mitchcrocker** [cs481-ekh/f22-kneed-for-speed#109] [General]
Sprint 2 Goal:
Be able to upload and parse both result files and geometry files. UI buttons and dials should function, as well as clean up general look. Javascript and HTML to be linted, as well as some unit tests added for CI.

Did we meet this goal:
No we did not meet our goal in terms of the result and geometry files, there was some back and forth with the sponsor that created more work. The dials got more functionality and we started on the painting of the heat map and have a good idea of what to do there. 

What we can improve on:
Definitely need to focus a majority of the effort into the parsing of the files in order to paint the heat map correctly. That hasn't had very much progress since the beginning of the project and could block us down the road. This will be done by adding more people to those tasks and splitting up the tasks/stories into smaller, more specific ones. 

Overall performance
We feel that this sprint could've gone better, there were extemporaneous situations that didn't add great progress and communication could've been improved. It did get the ball rolling with some crucial aspects of the projects and will allow us to focus on other parts that have had issues. There was also more research done than last sprint due to new technologies being introduced. 

Team capacity and velocity
We think we overestimated our velocity, capacity and size of the tasks. In this next sprint we plan on breaking up stories into multiple stories on the parsing side, that whole section of the project started as two stories but we realize now it probably could've been an epic in itself. We're still sticking with the 2 stories per person, just making those stories a little more manageable. 

Stories to be carried over:
#30 
#31 
#107 (#83)
#84 
#80 
#29

**Comment by dennischen01** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
